Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }

```

Comment:
**Suggestion**: Refactor the Even method to use proper Java naming conventions. Method names should start with a lowercase letter. Additionally, consider extracting the logic for printing numbers from an array into a separate method that both Even and the main method can use to reduce code duplication. <br> **Code Suggestion**: 
 ```
 -    public static void Even(String[] args) {
 +    public static void printEvenNumbers(String[] args) {
 ```